wsclient/pom.xml

```diff
@@ -46,6 +46,10 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.1</version>
+          <configuration>
+                    <source>1.6</source>
+                    <target>1.6</target>
+                </configuration>
```

Comment:
I didn't expect this change to require mvn build changes, what's the net affect of this pom changes?